test/e2e/run_test.sh

```diff
@@ -20,10 +20,24 @@ function setup_kind_cluster() {
     local NAME="${1:-kmesh-testing}"
     local IMAGE="${2:-"${DEFAULT_KIND_IMAGE}"}"
 
-    # Delete any previous KinD cluster.
-    echo "Deleting previous KinD cluster with name=${NAME}"
-    if ! (kind delete cluster --name="${NAME}" -v9) > /dev/null; then
-        echo "No existing kind cluster with name ${NAME}. Continue..."
+    local FORCE_RECREATE=false
```

Comment:
Also no need for logic of `FORCE_RECREATE`, keep `setup_kind_cluster` unchanged.